gen3/bin/kube-roll-all.sh

```diff
@@ -86,11 +86,23 @@ else
   gen3_log_info "no manifest entry for fence"
 fi
 
-if g3kubectl get cronjob etl >/dev/null 2>&1; then
-    gen3 job run etl-cronjob
+# Set a var for the cron folder path
+g3k_cron_manifest_folder="$(g3k_manifest_path | rev | cut -d '/' -f2- | rev)/manifests/cronjobs"
+# Check for file with defined cronjobs
+if [[ -f "$g3k_cron_manifest_folder/cronjobs.json" ]]; then
+  keys=$(g3k_config_lookup 'keys[]' $g3k_cron_manifest_folder/cronjobs.json)
```

Comment:
could you document the file location and expected format somewhere? :D

Reply:
Yea, I just wanted to push up the initial config first, but Ill document it with my next commit, which should also contain some other fixes.